File: modules/swagger-generator/src/test/scala/GeneratorInputTest.scala
import io.swagger.generator.model._
import org.junit.runner.RunWith
import org.scalatest.{FlatSpec, Matchers}
import org.scalatest.junit.JUnitRunner

@RunWith(classOf[JUnitRunner])
class GeneratorInputTest extends FlatSpec with Matchers {
  it should "write an object" in {
    val obj = new GeneratorInput()
    obj.setSwaggerUrl("http://petstore.swagger.io/v2/swagger.json")
  }
}

File: modules/swagger-generator/src/test/java/io/swagger/generator/GeneratorInputTest.java
package io.swagger.generator;

import io.swagger.generator.model.GeneratorInput;

import org.testng.annotations.Test;

@SuppressWarnings("static-method")
public class GeneratorInputTest {

    @Test(description = "write an object")
    public void writeObjectTest() {
        final GeneratorInput generatorInput = new GeneratorInput();
        generatorInput.setSwaggerUrl("http://petstore.swagger.io/v2/swagger.json");
    }
}
